libherokubuildpack/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "libherokubuildpack"
+version = "0.10.0"
+edition = "2021"
+rust-version = "1.60"
+license = "BSD-3-Clause"
+description = "Opinionated common code for buildpacks implemented with libcnb.rs"
+keywords = ["buildpacks", "CNB"]
+repository = "https://github.com/heroku/libcnb.rs"
```

Comment:
```suggestion
repository = "https://github.com/heroku/libcnb.rs/tree/main/libherokubuildpack"
```

Reply:
Fixed in [bdebdbf](https://github.com/heroku/libcnb.rs/pull/495/commits/bdebdbfa79bf3f7fe479d498b1d78b119a9bd7fc)